scapy/fields.py

```diff
@@ -1136,35 +1136,24 @@ def __init__(self, value, names):
         self.multi = isinstance(names, list)
         self.names = names
         self.value = self._fixvalue(value)
-    def __hash__(self):
-        return hash(self.value)
-    def __int__(self):
-        return self.value
-    def __eq__(self, other):
-        return self.value == self._fixvalue(other)
-    def __lt__(self, other):
-        return self.value < self._fixvalue(other)
-    def __le__(self, other):
-        return self.value <= self._fixvalue(other)
-    def __gt__(self, other):
-        return self.value > self._fixvalue(other)
-    def __ge__(self, other):
-        return self.value >= self._fixvalue(other)
-    def __ne__(self, other):
-        return self.value != self._fixvalue(other)
+    __hash__ = lambda self:  hash(self.value)
```

Comment:
I don't like using lambda functions here. Is there a good reason for that?

Reply:
Not really. It just takes way less space and is (to me) more readable.

I'm ok to remove them if needed.

Reply:
If there is no specific reason for that, then please remove this change.